### DISCUSSION TITLE
Using AWS Secret key from Vault

### DISCUSSION THREAD

**Discussion by vitali-federau-fivestars** [vmware-tanzu/velero#4830] [Community support Q&A]
Greetings, great Velero Community!
I have a question - I am using values.yaml + helm chart in order to create the velero server component,
In order to have the AWS secret key properly stored, I want to use Vault for storing secret,
Can I somehow reference the vault secret in my values.yaml file to reference the Vault?
Thanks for any help!
Best Regards

**Comment by KmZ78** [comment]
Do you have any news on this ?
